packages/perseus/src/styles/styles.less

```
@@ -70,11 +70,10 @@
         }
 
         // Inline math should have the same font-size as the surrounding text.
-        // There is no wrapping element around .katex to indicate that it is
+        // There is no wrapping element around math to indicate that it is
```

```suggestion
        // There is no wrapping element around TeX to indicate that it is
```
I wonder if this would be clearer to future humans, but feel free to disregard if you disagree